src/provision_server.py

```diff
@@ -150,12 +162,17 @@ def print_conf(D, path, owner = None, perm = None):
 	parts = parts.loc[~parts["start"].isna() & (~parts["partition"].isin(nonstandardparts))].astype({ "start" : int, "end" : int })
 
 	nonpreemptible_range = list(itertools.chain(*[range(x, y + 1) for x, y in parts.loc[parts["partition"].str.contains(r"-nonp$"), ["start", "end"]].values]))
+	gpu_range =  list(itertools.chain(*[range(x, y + 1) for x, y in parts.loc[parts["partition"].str.contains(r"-nonp-\d-.*"),["start", "end"]].values]))
```

Comment:
Not clear to me how this regex selects for GPU nodes. Review.

Reply:
gpu node names are of the form <machinetype>-nonp-<num_accelerators>-<accelerator_type> I don't think anyone would be crazy enough to use attach 10+ gpus, hence the single digit. This could be simplified to  r"-nonp-" instead, or a additional gpu flag could be added to the machine name